src/manage_sql.c

```diff
@@ -54467,31 +54467,35 @@ update_from_slave (task_t task, entity_t get_report, entity_t *report,
   current_commit_size = 0;
   while ((host_start = first_entity (hosts)))
     {
-      if (strcmp (entity_name (host_start), "host_start") == 0)
+      if (strcmp (entity_name (host_start), "host") == 0)
         {
-          entity_t host, end;
+          entity_t ip, end;
           char *uuid;
 
-          host = entity_child (host_start, "host");
-          if (host == NULL)
+          ip = entity_child (host_start, "ip");
+          if (ip == NULL)
+            goto rollback_fail;
+
+          start = entity_child (host_start, "start");
+          if (start == NULL)
             goto rollback_fail;
 
           end = entity_child (host_start, "end");
           if (end
               && entity_text (end)
               && strcmp (entity_text (end), "")
               && report_host_noticeable (current_report,
-                                         entity_text (host)))
+                                         entity_text (ip)))
             {
               uuid = report_uuid (current_report);
-              host_notice (entity_text (host), "ip", entity_text (host),
-                           "Report Host", uuid, 1, 1);
```

Comment:
Looks like a space was lost here.  Might want to put it back if you're trying to make it the same as master.

Reply:
Oh this was actually consistent with master (but not my slightly out of date master).  Anyway, thanks for fixing.